project-base/storefront/components/Pages/ErrorPage/Error500Content.tsx

```diff
@@ -29,9 +34,13 @@ export const Error500Content: FC = () => {
                 <ErrorPage isWithoutImage>
                     <ErrorPageTextHeading>{t('Something went wrong.')}</ErrorPageTextHeading>
                     <ErrorPageTextMain>{t('Please try again later or contact us.')}</ErrorPageTextMain>
-
                     <ErrorPageButtonLink href="/">{t('Back to shop')}</ErrorPageButtonLink>
                 </ErrorPage>
+                {err && (
+                    <div className="my-3 flex justify-center">
+                        <CopyTextBlock showCopyTextButton={false} textToCopy={err} />
```

Comment:
This `showCopyTextButton` prop is confusing me since here is already used CopyTextBlock which should by name already include this functionality. 

My proposal would be to put condition here and either show `CopyTextBlock` which will allow user to copy or simple div.

Reply:
Smart, let's do that. Just so you know, the reason why the button is not visible in some cases, is because Next.js does not allow button onClicks on the 500 page.